.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,14 @@
+# https://beta.ruff.rs
+name: lint_python
+on:
```

Comment:
I'd suggest copying the filters from https://github.com/nodejs/build/pull/3301/files#diff-14721317565b7c66355f276ec27762054b12f390afd6c35fba85cfaf77d311c7R2, but adding `pyproject.toml` to the lists

Reply:
It's not about tool speed, it's about not running jobs that can have unrelated job failures on a PR